pkg/api_builder/lib/src/client_library_generator.dart

```diff
@@ -131,6 +131,9 @@ code.Method _buildClientMethod(
     returnTypeRef = code.refer('List<int>');
   }
 
+  final queryMap =
+      h.queryParameters.map((p) => '\'${p.name}\': ${p.name}').join(', ');
```

Comment:
please add a comment that this is the serialized format of the query parameters map.